hack/init-environment.sh

```diff
@@ -32,6 +32,8 @@ declare -r HELM_OPERATOR_CHART_VERSION=0.4.0
 
 declare -r KUBECONFIG_DIR_NAME=.kubeconfig
 
+declare -r SECRET_MANAGER_VERSION=release-1.0.2
+declare -r SECRET_MANAGER_CHART_VERSION=0.0.4
```

Comment:
Nitpick: can you put these `declare`s in the same block as the ones for the Helm operator, so all the versions are grouped?